doc/kubernetes/modules/ROOT/pages/util/kc-failover.adoc

```diff
@@ -1,88 +1,65 @@
 = Simulate Keycloak Site Failover
-:description: How to automate the simulation of Keycloak site failure.
+:description: How to automate the simulation of Keycloak site failures.
 
 == Prerequisites:
 
-* A Keycloak instance replicated across two Openshift clusters with Infinispan xsite and an Aurora DB
-* Realm, user and client exist with the values required by the benchmark CLI command
+* A Keycloak instance replicated across two ROSA clusters with Infinispan xsite and an Aurora DB
+* xref:prerequisite/prerequisite-task.adoc[Task]
 
 == Running the failure test from the CLI
 
-=== Preparations
 
-* Extract the `+keycloak-benchmark-${version}.[zip|tar.gz]+` file
-* xref:benchmark-guide::preparing-keycloak.adoc[]
-* Make sure your local KUBECONFIG is set to the Openshift cluster which you want to fail.
+== Installation steps
 
-=== Parameters
+. Check out the Git repository from https://github.com/keycloak/keycloak-benchmark/.
 
-The failover scripts requires the following env variables to be set; `FAILOVER_MODE` and `DOMAIN`.
+. Change to the folder `link:{github-files}/provision/rosa-cross-dc/[provision/rosa-cross-dc]`.
+
+. Provide the <<required-variables>> in the `.env` file in the folder.
```

Comment:
I could not see this `required-variables` anchor resolving in my local. Am I missing a step for this to reflect properly?

![Screenshot 2024-09-09 at 16 23 40](https://github.com/user-attachments/assets/fb2c8ff7-62a9-45e6-944b-99d9d24b81e1)

Reply:
This was a leftover from copying and pasting from `cross-site-rosa.adoc`. I have remove the anchor :+1: